vendor/modules.txt

```diff
@@ -2549,6 +2550,7 @@ sigs.k8s.io/yaml
 # github.com/google/shlex => github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 # github.com/google/uuid => github.com/google/uuid v1.1.2
 # github.com/googleapis/gax-go/v2 => github.com/googleapis/gax-go/v2 v2.0.5
+# github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.5.1
```

Comment:
this was just moved to github.com/google/gnostic in https://github.com/kubernetes/kubernetes/pull/108644 (specifically the v0.5.7-v3refs revision to avoid tons of transitive dependencies in https://github.com/google/gnostic/issues/317) ... can we avoid reintroducing two names / versions for the same dependency?

Reply:
cc @Jefftree

Reply:
Thank you, and I'm sorry for missing this. We'll make the same change on our side (https://github.com/kubernetes-sigs/kustomize/pull/4538), release again, and update this PR to use the fixed versions.